src/components/common/UploadDocumentEwallet.tsx

```
@@ -1,108 +1,442 @@
 	};
-	const sendMessageToIframe = () => {
-		const jwtToken = localStorage.getItem('authToken');
-		if (iframeRef.current) {
-			iframeRef.current.style.display = 'block'; // Show iframe when button is clicked
-
-			iframeRef.current.contentWindow.postMessage(
-				{ type: 'JWT_TOKEN', payload: jwtToken }, // Message data
-				VITE_EWALLET_IFRAME_SRC // Replace with the iframe app's domain
+
+	// Prepare payload for document upload
+	const preparePayload = async (data: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)